### DISCUSSION TITLE
Craftable on existing projects

### DISCUSSION THREAD

**Discussion by Dambar93** [BRACKETS-by-TRIAD/craftable-pro-docs#69] [Q&A]
I got a question about Craftable PRO, there gonna be no problems in compabibility with projects which already exist and instaling on them craftable? I not gonna get any errors? Because we wanna buy license and use it on ours existing projects. And if there is refund, if we don't like it?

**Comment by palypster** [comment]
Hello, I'd like to kindly ask you to send an email to support@craftable.pro. Thank you.
